README.md

```diff
@@ -585,6 +585,16 @@ To claim your rewards, run the following command:
 ./scripts/claim-bot-rewards.sh <PATH_TO_YOUR_PAYMENT_SIGNING_KEY> <PATH_TO_YOUR_CARDANO_NODE_SOCKET>
 ```
 
+## Monitoring
+There are several way you could keep an eye on the performance of your SOR instance.
+- You could look up the address of your SOR on any blockchainexplorer like [cexplorer](https://cexplorer.io/) or [cardanoscan](https://cardanoscan.io/) and keep an eye on the transactions manually.
+- You could automate this by using the [Thoth Cardano Bot](https://github.com/DevStakePool/thoth-bot) for Telegram, that you can configure to send you notifications every time your SOR is able to succesfully submit a match and you are the lucky one to be the one who is getting the rewards.
+- Or just use the community [developed script](https://github.com/cloudstake/public/blob/main/genius-yield-errors.sh).
```

Comment:
Regarding this community developed script, it's matching upon log messages & we might change our log formatting, etc. down the line breaking this script. Do we want to either omit it or mention that it's not maintained by us & can be out of sync?

Reply:
I can mention that it is not maintained by us. I think it is obvious, since it is "community developed", but an explicit mention of this fact won't hurt.

I have added an explicit mention about the fact that this script is not maintained by us: a74a86c41476d2308ea8a81677c40d5c069aa871